target/linux/mxs/image/Makefile

```diff
@@ -46,8 +46,8 @@ endef
 define Device/i2se_duckbill
   DEVICE_VENDOR := I2SE
   DEVICE_MODEL := Duckbill
-  DEVICE_PACKAGES := -dnsmasq -firewall -ppp -ip6tables -iptables -6relayd -mtd \
-		     uboot-envtools kmod-leds-gpio -kmod-ipt-nathelper
+  DEVICE_PACKAGES := -dnsmasq -firewall4 -ppp -ip6tables -iptables -mtd \
```

Comment:
This is a bit strange since we later on try to install LuCI, which pull ins firewall again.

Also typo here, it's mxs not msx :)

Reply:
Context https://buildbot.aparcar.org/phase1/#/builders/86/builds/16/steps/37/logs/stdio